### PR TITLE
Update to 10.2.0

### PR DESCRIPTION
**Pull request by cbouss**
Changes:
- Update to 10.2.0

https://github.com/python-pillow/Pillow/tree/10.2.0

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -34,15 +34,15 @@ requirements:
     - freetype 2.10.4
     - lcms2 2.12       # [not win]
     - libtiff 4.2.0
-    - libwebp 1.3.2
-    - openjpeg
+    - libwebp-base 1.3.2
+    - openjpeg 2.3
     - tk 8.6.12
```
Comment (JeanChristopheMorinPerso) [inline recipe/meta.yaml:39]: conda-build says that tk isn't used. From what I see, it uses an embedded copy of tk, see https://github.com/python-pillow/Pillow/blob/10.2.0/setup.py#L983 (`src/Tk`).

**Review by skupr-anaconda — APPROVED** [review]
LGTM but JC's comment

File: recipe/meta.yaml (hunk 1 above)
Comment (cbouss) [reply recipe/meta.yaml:39]: Based on this, I feel like removing the tk dependency from the recipe and adding license TCL to the list of licenses. Do you agree?

Comment (JeanChristopheMorinPerso) [reply recipe/meta.yaml:39]: I think it would make sense. Or we could patch the code to use our Tk. But it's probably easier to just do what you said.

Comment (cbouss) [reply recipe/meta.yaml:39]: I will remove the dependency. Given They only include a subset of one file of tk.

**Review by JeanChristopheMorinPerso — APPROVED** [review]

**Pull request merged**